nilearn/input_data/nifti_spheres_masker.py

```diff
@@ -73,11 +110,16 @@ def _apply_mask_and_get_affinity(seeds, niimg, radius, allow_overlap,
     mask_coords = mask_coords.astype(int).tolist()
     for i, seed in enumerate(seeds):
         try:
-            A[i, mask_coords.index(seed)] = True
+            A[i, mask_coords.index(list(map(int, seed)))] = True
         except ValueError:
             # seed is not in the mask
             pass
 
+    sphere_sizes = np.asarray(A.tocsr().sum(axis=1)).ravel()
+    empty_spheres, *_ = np.nonzero(sphere_sizes == 0)
```

Comment:
```suggestion
    empty_spheres, _ = np.nonzero(sphere_sizes == 0)
```

Reply:
I don't think this change would work, `nonzero` returns a tuple with one element per dimension (so 1 here)
It could be also `empty_spheres = np.nonzero(sphere_sizes == 0)[0]`

Reply:
I will use the `empty_spheres = np.nonzero(sphere_sizes == 0)[0]`, feels more intuitive for me.